pkg/operator/ceph/cluster/cluster.go

```diff
@@ -397,6 +411,66 @@ func (c *ClusterController) preClusterStartValidation(cluster *cluster, clusterO
 	return nil
 }
 
+func (c *cluster) removeDefaultReplicationRule() error {
+	err := c.context.Executor.ExecuteCommand("ceph", "osd", "crush", "rule", "rm", "replicated_rule")
+	if err != nil {
+		if exitErr, ok := err.(*exec.ExitError); ok && exitErr.ExitCode() == int(syscall.EBUSY) {
```

Comment:
I'd suggest calling the rook helper `exec.ExitStatus(err)` to get the error code. Or you could add another helper method next to it if needed. It will just help simplify the error handling code a bit.